components/script/dom/writablestreamdefaultcontroller.rs

```diff
@@ -389,6 +396,12 @@ impl WritableStreamDefaultController {
         self.underlying_sink_obj.set(*this_object);
     }
 
+    /// <https://streams.spec.whatwg.org/#writablestreamdefaultcontroller-abortcontroller>
+    pub(crate) fn abort_controller(&self) -> DomRoot<AbortController> {
+        // The abort controller getter steps are to return this’s [[abortController]].
+        self.abort_controller.as_rooted()
```

Comment:
Where is this from in the spec? I think you can just call this method `signal_abort`, since that is how it is used.